docker/docker/src/main/java/org/arquillian/cube/docker/impl/util/AutoStartOrderUtil.java

```diff
@@ -70,6 +86,14 @@
         return new HashSet<>(nodes.values());
     }
 
+    private static boolean isRegularExpressionBased(String[] autoStartContainers) {
+        return autoStartContainers.length == 1;
```

Comment:
what if you only have one named container to start? Any chance a container name could cause a 'fail to parse regexp' or create a regexp that doesn't actually match the name?